controllers/workloads/console/controller.go

```diff
@@ -660,14 +841,19 @@ func (r *ConsoleReconciler) buildJob(logger logr.Logger, name types.NamespacedNa
 		jobTemplate.ObjectMeta.Labels,
 	)
 
+	podTemplate := (*corev1.PodTemplateSpec)(jobTemplate)
+	if r.EnableSessionRecording {
+		podTemplate = r.modifyPodTemplate(logger, podTemplate)
```

Comment:
`modifyPodTemplate` is arguably a bit of a generic name, given that it's only relevant within the context of session recording

Reply:
👍